src/main/java/org/fundacionjala/pivotal/ui/pages/SideBarBoard.java

```diff
@@ -0,0 +1,24 @@
+package org.fundacionjala.pivotal.ui.pages;
+
+import org.fundacionjala.pivotal.ui.pages.common.CommonActions;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * Created by bruno on 4/9/2017.
```

Comment:
you don't need to commit your name and the date in the comments.
Please remove it in all your classes